packages/driver/cypress/e2e/commands/aliasing.cy.ts

```diff
@@ -40,21 +40,23 @@ describe('src/cy/commands/aliasing', () => {
       const li = cy.$$('#list li').eq(0)
 
       cy.get('#list li').eq(0).as('firstLi').then(($li) => {
-        expect($li).to.match(li)
+        expect($li[0]).to.eq(li[0])
       })
     })
 
     it('retries previous commands invoked inside custom commands', () => {
-      Cypress.Commands.add('get2', (selector) => cy.get(selector))
+      // @ts-expect-error TODO: add types need updating to not error with string here
+      Cypress.Commands.add('get2', (selector: string) => cy.get(selector))
```

Comment:
Our Cypress.Commands.add types are not ideal. I tried to update it but it broke multiple other things.